### DISCUSSION TITLE
can I register for updates to the "discussions" page?

### DISCUSSION THREAD

**Discussion by jpl33** [Security-Onion-Solutions/securityonion#1793] [General] [answered]
like we did to the Google group for previous versions?
it was very effective for me, and I guess the rest of the SO community/

**Comment by TOoSmOotH** [comment]
It looks like you can subscribe to individual topics. Will check into discussions as a whole.

**Comment by dougburks** [comment, marked as answer]
I think you may be able to do this by "watching" the repo. Try clicking the Watch button in the upper right corner of the page.
